tests/System.Text.Primitives.Tests/Encoding/PerformanceTests.cs

```diff
@@ -251,7 +251,7 @@ public unsafe void ReverseEnumerateCodePointsConstructFromSpan()
                         Utf8String.CodePointReverseEnumerator it = utf8s.CodePoints.GetReverseEnumerator();
                         while (it.MoveNext())
                         {
-                            UnicodeCodePoint codePoint = it.Current;
+                            var codePoint = it.Current;
```

Comment:
We don't use var unless it's super clear what the type is. Here it does not seem to be super clear.